shared/downloadAndVerifyFileIntegrity.ts

```diff
@@ -0,0 +1,76 @@
+import { readFile } from "fs-extra";
+import { createHash } from 'crypto';
+import retry from 'async-retry';
+import { DownloadChannels } from "../src/@types/ipc_channels";
+import { downlaodFileToDest } from "./downloadFileToDest";
+import Logger from "./logger";
+
+const DEFAULT_RETRIES = 5;
+
+  export async function getSumsHashFromFile(sumsFilePath: string) {
```

Comment:
Please fix indentation here and below. Also the function name is not very clear, and looks like it gets a `hash` while it actually gets `hashes`. How about renaming it to `getHashsumsFromFile` or `getChecksumsFromFile`? Please let me know if you have any objections.